datadog/api/exceptions.py

```diff
@@ -3,7 +3,20 @@
 """
 
 
-class ProxyError(Exception):
+class DatadogException(Exception):
+    """
+    Base class for Datadog API exceptions.  Use this for patterns like the following:
+
+        try:
+            # do something with the Datadog API
+        except datadog.api.exceptions.DatadogException:
+            # handle any Datadog-specific exceptions
+    """
+    def __init__(self, *args, message='', **kwargs):
```

Comment:
Empty message as default for an exception is not a good idea.

If you want to force having a message I'd prefer having it the way it was before then.

Reply:
ok, no worries :) i've removed the `__init__()` (turns out i had introduced a bug in my last commit anyway)